deploy/operator.yaml

```diff
@@ -45,15 +45,15 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
-              value: "nmstate-handler"
+              value: "{{with .HandlerPrefix}}{{printf "%s-" .}}{{end}}nmstate-handler"
             - name: NODE_NAME
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
             - name: INTERFACES_FILTER
               valueFrom:
                 configMapKeyRef:
-                  name: nmstate-config
+                  name: {{with .HandlerPrefix}}{{printf "%s-" .}}{{end}}nmstate-config
                   key: interfaces_filter
             - name: WEBHOOK_PORT
               value: "54874"
```

Comment:
More of a question, do we have to ensure that this port does not collide (the pod is running with network=host to be able to configure node networking) with the webhook installed by the other operator ? or is webhook going to be not run until we undeploy the other (with the lock PR).

Reply:
We can parameterize it if we want. You tell me.

Reply:
Nah let's skip it we will see.